exercise_1-3.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta name="keywords" content="HTML">
+		<meta name="description" content="Image, Video and Sound">
+		<meta name="author" content="Juan Cruz Yezzi">
+		<meta charset="utf-8">
+		<title>Topic 0</title>
+	</head>
+	<body>
+		<img src="http://im.ziffdavisinternational.com/ign_es/screenshot/default/alamuerte_wju7.png">
+		<video src="Hearthstonevideo.webm" controls></video>
+		<audio src="Soundtrack.mp3" controls></audio>
```

Comment:
on this tag will be better to have the workarround if the browser don't support that tag functionality